### ISSUE TITLE
Feature Request: Migrate from gym to gymnasium

### ISSUE THREAD

**Issue by elliottower** [Talendar/flappy-bird-gym#7]
Hi, would it be possible for flappy-bird-gym to be upgraded from gym to gymnasium? Gymnasium is the maintained version of openai gym and is compatible with current RL training libraries ([rllib](https://github.com/ray-project/ray/tree/master/rllib) and [tianshou](https://github.com/thu-ml/tianshou) have already migrated, and [stable-baselines3](https://github.com/DLR-RM/stable-baselines3/blob/e5deeed16efb57c34ccdcb14692439154d970527/docs/guide/install.rst#bleeding-edge-version) will soon).

This is currently listed in the gymnasium [third party environments](https://gymnasium.farama.org/environments/third_party_environments/) but we are cleaning the list up to only include maintained gymnasium-compatible repositories.

For information about upgrading and compatibility, see [migration guide](https://gymnasium.farama.org/content/migration-guide/) and [gym compatibility](https://gymnasium.farama.org/content/gym_compatibility/).

**Comment by elliottower** [comment]
Looks like this branch has done the migration: https://github.com/markub3327/flappy-bird-gymnasium, is there any plans to merge this into master? Looks like it is being actively updated by @markub3327

**Comment by markub3327** [comment]
@elliottower No, this is a separate branch only for Gymnasium.

**Comment by elliottower** [comment]
> @elliottower No, this is a separate branch only for Gymnasium.

Makes sense, just figured I’d ask in case it was missed/not on your radar. Cheers.

**Comment by elliottower** [comment]
@Talendar would it be possible to link to the [Gymnasium fork](https://github.com/markub3327/flappy-bird-gymnasium) on the main README for this repository?
